Cooper.ORM/DbTools.cs

```diff
@@ -48,89 +19,191 @@ public static string GetVariableAttribute(string attribute)
+        }
+
+        public static string CreateDeleteQuery(string table, WhereRequest whereRequest = null)
+        {
+            return string.Format("DELETE FROM {0} {1}", table, whereRequest?.ToString());
+        }
+
+        public static string SanitizeString(string value)
+        {
+            if (value == null) { 
+                return null;
+            }
+            return value.Replace("'", "").Replace("\"", "");
+        }
+
+        public static bool ProcessBoolean(object value)
+        {
+            if (value == null)
+            {
+                return false;
+            }
+
+            return (value.ToString() == "y") ? true : false;
         }
 
-        public static string CreateUpdateQuery(string table, EntityORM entity, DbTools.WhereRequest[] whereRequests = null)
+        public static string WrapBoolean(bool value)
         {
-            string where = CreateWhereRequest(whereRequests);
-            return String.Format("UPDATE {0} SET {1}{2}", table, string.Join(",", entity.attributeValue.Select(x => x.Key + "=" + x.Value).ToArray()), where);
+            return (value) ? "\'y\'" : "\'n\'";
         }
 
-        public static string CreateDeleteQuery(string table, WhereRequest[] whereRequests = null)
+        public static string Wrapstring(string value)
         {
-            string where = CreateWhereRequest(whereRequests);
+            return string.Format("\'{0}\'", value);
+        }
+    }
 
-            return String.Format("DELETE FROM {0} {1}", table, where);
+    public class WhereRequest
+    {
+        private StringBuilder whereRequest = new StringBuilder();
+
+        private static readonly string[] operators = new string[] { "=", ">=", "<=", ">", "<", "IS NULL", "IS NOT NULL", "IN" };
+
+        private static string GetOperatorstring(Operators requestOperator)
+        {
+            return operators[(int)requestOperator];
         }
 
-        private static string CreateWhereRequest(WhereRequest[] whereRequests = null)
+        public WhereRequest(string attributeName, Operators op, params string[] attributeValues)
         {
-            string where = "";
-            if (whereRequests != null)
+            whereRequest.Append(attributeName);
+            if (op == Operators.Null || op == Operators.NotNull)
+            {
+                whereRequest.Append($" {GetOperatorstring(op)}");
+            }
+            else if (attributeValues.Length > 0)
             {
-                where = " WHERE";
-                foreach (WhereRequest request in whereRequests)
+                if (op == Operators.In)
+                {
+                    whereRequest.Append(" IN ");
+                    whereRequest.Append("(");
+                    whereRequest.Append(string.Join(", ", attributeValues));
+                    whereRequest.Append(")");
+                }
+                else if (attributeValues.Length == 1)

```

Comment:
not needed case. Let it go by main if. So it will be attr IN (3), not a problem

Reply:
You're right. I'll change it.